admin/client/src/boot/index.js

```diff
@@ -68,6 +68,10 @@ function appBoot() {
   // Bootstrap routing
   const routes = new BootRoutes(store);
   routes.start(window.location.pathname);
+
+  // @TODO - Remove once we remove entwine
+  // Enable top-level css selectors for react-dependant entwine sections
+  window.jQuery('body').addClass('react-boot');
```

Comment:
please can we use a `.js-` prefix on this?

http://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/#javascript-namespaces-js-